keystrokes/src/components/utils/utils.js

```diff
@@ -1,5 +1,53 @@
 export function getNextKey(keys) {
-	return keys[
-		Math.floor(Math.random() * keys.length)
-	];
+	return keys[Math.floor(Math.random() * keys.length)];
+}
+
+export function verifyKey(key) {
+	if (key.length > 2) {
+		if (key === 'backquote') {
```

Comment:
this looks like a good use case for a switch statement but nbd

Reply:
made it into a dictionary to follow the design of the new validate press